icicle/include/api/babybear/field/api_ext.h

```diff
@@ -0,0 +1,21 @@
+// WARNING: This file is auto-generated by a script.
```

Comment:
why split ext?

Reply:
Just trying to follow the same structure we have in the codebase. But I guess we can merge in a single file, fine